build.sbt

```diff
@@ -79,6 +79,24 @@ lazy val generated = project
   .settings(scalacOptions := Seq())
   .setupBuildInfo
 
+lazy val models = project
+  .in(file("models"))
+  .settings(
+    name                := "interop-be-authorization-management-models",
+    scalacOptions       := Seq(),
```

Comment:
Can you remove it? There's no reason not to have all the benefits of tpolecat, even if the module is just a bunch of case classes. In a previous service, I noticed some unused imports.